smoke-tests/designer/features/pageobjects/pages/addComponent.page.js

```diff
@@ -2,7 +2,7 @@ const Page = require("./basePage");
 
 class AddComponentPage extends Page {
   get parent() {
-    return $(".flyout__container");
+    return browser.$(".panel--flyout");
```

Comment:
feel free to use and add `data-test-id` to elements (no need to change this now). They're least likely to be missing/changed without affecting your tests vs a class